tests/Endpoints/DocumentsTest.php

```
@@ -429,17 +411,16 @@ public function testAddDocumentsNdjsonInBatches(): void
     public function testAddWithUpdateDocuments(): void
     {
         $index = $this->createEmptyIndex($this->safeIndexName('movies'));
-        $response = $index->addDocuments(self::DOCUMENTS);
-        $index->waitForTask($response['taskUid']);
+
+        $task = $index->addDocuments(self::DOCUMENTS)->wait();
```

PHPMD (2.15.0)

Avoid unused local variables such as '$task'. (undefined)

(UnusedLocalVariable)